README.md

```diff
@@ -4,6 +4,29 @@
 
 🔧 Settings for Loadsmart's hosted Danger instance
 
+## How to add Peril to my repo
+
+**Note**: this applies to repositories under Loadsmart organization. If
+you want to install peril for your repositories, you probably want to
+take a look at the general instructions available at
+[peril repository][peril-repo].
+
+For Loadsmart organization, the app is already instaled and configured.
+You'll only need to enable it for your repository.
+
+Go to [org settings][org-settings], and then select _Installed Github
+Apps_.
+
+Choose the one named `peril-loadsmart`:
+
+![peril-loadsmart app](images/choose-app.png)
```

Comment:
I can't see the image, maybe it is just because github's preview is not able to show it.

Reply:
Never mind, it loaded now, sorry

Reply:
If you go to the file on the branch, you can see it: https://github.com/loadsmart/peril-settings/blob/bf944ef9d444a4ddc8f9a36c3ef019ea13229114/README.md